controllers/auth.js

```diff
@@ -12,7 +12,7 @@ const authService = require("../services/authService");
 const githubAuth = (req, res, next) => {
   let userData;
   const rdsUiUrl = new URL(config.get("services.rdsUi.baseUrl"));
-  const authRedirectionUrl = req.query.state ?? rdsUiUrl;
+  let authRedirectionUrl = req.query.state ?? rdsUiUrl;
```

Comment:
why did we change const to let over here?

Reply:
we can change the redirection URL if we have incomplete user details, so we can redirect them to the new sign up flow